themes-default/slim/src/components/display-show.vue

```diff
@@ -504,6 +517,36 @@ export default {
                     $('#' + seasonNo + '-cols').show();
                 }
             });
+        },
+        jumpToSeason(event) {
+            const { id } = event.currentTarget;
```

Comment:
That is incorrect, it uses `#jumpToSeason` which is the `<select>` id, not the target id.

Reply:
I'm confused as this works...